### PR TITLE
INTERFACES-226: implemented new validator list with infinite scroll i…

### PR DESCRIPTION
**Pull request by axelrod-blox**
…nstead the old one with pagination

### CODE REVIEW

File: src/app/components/applications/SSV/MyAccount/components/Validator/SingleCluster/components/Settings/Settings.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ import { getBeaconChainLink, isMainnet } from '~root/providers/networkInfo.provi
 import { SingleCluster, BULK_FLOWS } from '~app/model/processes.model';
 import { ENABLE_EXIT_FLOW, getLocalStorageFlagValue } from '~lib/utils/developerHelper';
 
-const Settings = ({ validator }: { validator: any }) => {
+const Settings = ({ validator, withoutSettings }: { validator: any, withoutSettings?: boolean }) => {
```
Comment (IlyaVi) [inline src/app/components/applications/SSV/MyAccount/components/Validator/SingleCluster/components/Settings/Settings.tsx:16]: { validator: any **;** withoutSettings?: boolean }

File: src/app/components/applications/SSV/MyAccount/components/Validator/ValidatorsList/ValidatorsList.tsx [hunk 2]
```diff
@@ -145,7 +169,11 @@ const ValidatorsList = ({
       }) => validator.validator));
     } else {
       validatorsByClusterHash(1, getClusterHash(cluster.operators, walletStore.accountAddress), clusterValidatorsPagination.rowsPerPage).then((response: any) => {
-        setClusterValidators(response.validators);
+        if (response.validators && response.validators.length) {
+          setClusterValidators(response.validators);
+        } else {
+          setNoValidatorsData(true);
```
Comment (IlyaVi) [inline src/app/components/applications/SSV/MyAccount/components/Validator/ValidatorsList/ValidatorsList.tsx:175]: use undefined or null as initial value for **clusterValidators** 
this way setting **[]** will have same effect as `setNoValidatorsData(true);`

**Review by IlyaVi — APPROVED** [review]

File: src/app/components/applications/SSV/MyAccount/components/Validator/ValidatorsList/ValidatorsList.tsx (hunk 2 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/MyAccount/components/Validator/ValidatorsList/ValidatorsList.tsx:175]: initial value for clusterValidators is empty array []

Comment (axelrod-blox) [reply src/app/components/applications/SSV/MyAccount/components/Validator/ValidatorsList/ValidatorsList.tsx:175]: no validator flag should trigger only if we have undefined or empty array of validators in response from API

File: src/app/components/applications/SSV/MyAccount/components/Validator/SingleCluster/components/Settings/Settings.tsx (hunk 1 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/MyAccount/components/Validator/SingleCluster/components/Settings/Settings.tsx:16]: any type?

**Review by alexstolr — APPROVED** [review]

**Comment by axelrod-blox** [comment]
Approved by Bunin
Can be merged

**Pull request merged**